### ISSUE TITLE
Cannot attach error handler

### ISSUE THREAD

**Issue by felixrabe** [gulp-community/gulp-pug#33]
This does not call `handleError`, causing `gulp watch` to break on every syntax error:

```
var gutil, handleError, jade;

gutil = require('gulp-util');
jade = require('gulp-jade');

function handleError(err) {
  gutil.log('handleError called!!!');
  this.emit('end');
};

gulp.task('build', function() {
  return gulp.src('./input/**/*.jade')
    .pipe(jade().on('error', handleError))
    .pipe(gulp.dest('./output'));
});

gulp.task('watch', function() {
  return gulp.watch(['./input/**/*'], ['build']);
});
```

**Comment by felixrabe** [comment]
This is with `gulp-jade` version 0.6.0.

**Comment by felixrabe** [comment]
Plugins that work for me as expected (and do not break my `watch` workflow) that you might want to look at for a reference:
- https://github.com/wearefractal/gulp-coffee
- https://github.com/plus3network/gulp-less

**Comment by felixrabe** [comment]
Versions in use:

```
$ npm ls --depth=0
@ /.../the-project
├── coffee-script@1.7.1
├── gulp@3.8.1
├── gulp-coffee@2.0.1
├── gulp-jade@0.6.0
├── gulp-less@1.2.3
├── gulp-util@2.2.17
└── rimraf@2.2.8
```

**Comment by phated** [comment]
I'm not going to have time to look into this any time soon.  If you want to do the legwork and figure out what your issue is, I will be happy to merge a fix.

**Comment by phated** [comment]
There are tests for error handling that have always worked, but note that @tomchentw updated the error pattern and it has been published as `0.6.1` - please update and test your code. Hopefully it solves your issue.

**Comment by phated** [comment]
Closing due to inactivity.
